lib/suite_runner.js

```diff
@@ -92,8 +92,11 @@ function runTestProcess(childProcess, debugPort, timeout, slowThreshold, reporte
     [testInterfacePath, JSON.stringify(parameters), testPath.file].concat(testPath.path),
     { silent: true, execArgv: process.execArgv.concat(debugArgs) });
 
-  hookStream(testPath, child.stdout, reporter, 'stdout');
-  hookStream(testPath, child.stderr, reporter, 'stderr');
+  // might be null in case of listening for debugger
+  if (child.stdout && child.stderr) {
+    hookStream(testPath, child.stdout, reporter, 'stdout');
+    hookStream(testPath, child.stderr, reporter, 'stderr');
+  }
```

Comment:
Would it make sense to have one if statement per line here? Are you sure that stdout and stderr always will exist or not exist both at the same time?